python/lsst/ap/verify/measurements/api.py

```diff
@@ -0,0 +1,64 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+"""Code for measuring software performance metrics.
```

Comment:
I couldn't think of one, and still can't. This file's purpose is to provide a single point of entry to the `measurements` sub-package, so that `ap_verify.py` doesn't need to know about implementation details like "which measurements have actually been implemented"?

Is there a name that communicates that to you?